ManagementFrontendDockerfile

```diff
@@ -1,4 +1,5 @@
-FROM frolvlad/alpine-gxx
+ARG CODE_VERSION
```

Comment:
Give this arg a default version (eg https://github.com/ucbrise/clipper/blob/develop/PythonContainerDockerfile#L1), otherwise the automated builds on Docker Hub will fail.